src/components/Updates/FinancialCard.css

```diff
@@ -0,0 +1,24 @@
+.ui.card.FinancialCard {
```

Comment:
I think you can avoid using `.ui.card` with some `!important`s. That is the way 🙏